docs/source/torchvision/datasets.rst

```diff
@@ -1,129 +1,77 @@
 torchvision.datasets
 ====================
 
-The following dataset loaders are available:
-
--  `MNIST`_
--  `COCO (Captioning and Detection)`_
--  `LSUN Classification`_
--  `ImageFolder`_
--  `Imagenet-12`_
--  `CIFAR10 and CIFAR100`_
--  `STL10`_
+All the datasets are subclasses of :class:`torch.utils.data.Dataset`
+i.e, they have ``__getitem__`` and ``__len__`` methods implemented.
+Hence, they can all be multi-threaded (python multiprocessing) using
```

Comment:
It's actually from [Readme](https://github.com/pytorch/vision/blob/master/README.rst) of pytorch/vision. Nevertheless clarified it.